osu.Game.Rulesets.Taiko/Judgements/TaikoDrumRollTickJudgement.cs

```diff
@@ -13,10 +13,21 @@ protected override int NumericResultFor(HitResult result)
         {
             switch (result)
             {
+                case HitResult.Great:
+                    return 200;
                 default:
                     return 0;
+            }
+        }
+
+        protected override double HealthIncreaseFor(HitResult result)
+        {
+            switch(result)
+            {
                 case HitResult.Great:
-                    return 200;
+                    return 0.0000003;
```

Comment:
Soo... 10,000,000 ticks award the same HP as 1 hit?

Reply:
it's the same in the old code